src/utils/validators.js

```diff
@@ -1,7 +1,7 @@
 
 // Common regexes and constants
 // ==============
-const PERMALINK_REGEX = '^(/([a-z]+([-][a-z]+)*/)+)$';
+const PERMALINK_REGEX = '^(/([a-z]+([-][a-z]+)*)+)$';
```

Comment:
regex should be `^(/([a-z]+([-][a-z]+)*/)+)$` to account for the ending slash

Reply:
Ah actually it should be `^(/([a-z]+([-][a-z]+)*)+/)$` right because we only want the forward slashes to be at the start and end

Reply:
But that would not allow a permalink like `/abc/123/` where there are forward slashes in between - is that the desired behavior?

Reply:
Currently, the PageSettingsModal conducts checks across all simple-pages to prevent repeated permalinks. However, if we allow a permalink like `/abc/123/`, there could be a chance that there exists another file in a different directory, for example collection `abc` with a file `123` that has a permalink `/abc/123`.

Preventing forward slashes will prevent such errors from happening. Thoughts?

Reply:
That makes sense, let's stick to your change `^(/([a-z]+([-][a-z]+)*)+/)$`. This regex makes sense for collections too since we'll probably want to pre-fill the collection part of the permalink for collection pages (meaning for collection **abc** the permalink `/abc/` part is pre-filled and non-editable)